src/GraphQL/Types/IGraphType.cs

```diff
@@ -1,14 +1,30 @@
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// A type that has a name within the GraphQL schema
+    /// </summary>
     public interface INamedType
     {
+        /// <summary>
+        /// Gets or sets a type name within the GraphQL schema.
+        /// Type names are case sensitive and consist of alphanumeric characters and underscores only. Type names cannot start with a digit.
+        /// </summary>
         string Name { get; set; }
     }
 
+    /// <summary>
+    /// Represents a graph type with the GraphQL schema.
+    /// </summary>
     public interface IGraphType : IProvideMetadata, INamedType
     {
+        /// <summary>
+        /// Gets or sets the description of the graph.
+        /// </summary>
         string Description { get; set; }
 
+        /// <summary>
+        /// Gets or sets the deprecation reason of the graph.
+        /// </summary>
         string DeprecationReason { get; set; }
```

Comment:
Hah! This property is not in the specification. It seems to have been defined here by mistake.